index.html

```diff
@@ -26,7 +26,7 @@
           </a>
         </nav>
       </div>
-      <div id="map"></div>
+      <div id="map" tabindex="0"></div>
```

Comment:
Why did you add the tabIndex? I would remove it to make the map navigable without previous selection.

Reply:
without we can't have keyboard event (keyboard pan or delete)

Reply:
I found a solution rtfm : https://github.com/geops/openlayers-editor/pull/201/commits/6e66d4896639f6f854e86ae6ce88d03f6ddf7598